app/dts/bindings/behaviors/zmk,behavior-toggle-layer.yaml

```diff
@@ -6,3 +6,13 @@ description: Toggle Layer
 compatible: "zmk,behavior-toggle-layer"
 
 include: one_param.yaml
+
+properties:
+  toggle-mode:
+    type: string
+    required: false
+    default: "on-off"
+    enum:
+      - "on"
+      - "off"
+      - "on-off"
```

Comment:
I think a term like "invert" might be a better name than "on-off". The latter is a bit ambiguous that it "toggles"/flips the state.

Reply:
I think invert could end up being confusing too. How about "switch"?

Reply:
I'd say "toggle" is the best but in this case it might not be a good idea 😄 Switch sounds OK too, or "flip".

Reply:
I like flip, made that change.